Documentation/contributing/development/reviewers_committers/triage.rst

```diff
@@ -0,0 +1,153 @@
+    time of writing, know about the requirement to resolve discussions, so it
+    may flag PRs as ready-to-merge even when you can't click the merge button.
+    In this case, review the discussions and see if there are any outstanding
+    discussions, and if not, close them.
+
+PRs that may need attention
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Cilium community contributions (open, unassigned, not draft)
+<https://github.com/cilium/cilium/pulls?q=is%3Aopen+is%3Apr+no%3Aassignee+draft%3Afalse+label%3Akind%2Fcommunity-contribution+sort%3Aupdated-asc+-author%3Aapp%2Frenovate>`_
+
+
+*   Work your way forward from the oldest-updated PRs. PRs with recent
+    activity are likely being interacted with by other team members.
+*   Apply ``kind/*`` and ``release-note/*`` labels, external contributors
+    cannot apply labels.
+*   Approve workflows and run ``/test``.
+*   Re-run failed jobs due to flakes (e.g. connecting to 1.1.1.1, ..). Link or
+    create issues for persistent flakes.
+*   Bump the PR if feedback was positive but there's been no progress, either
+    from reviewer or author side.
+*   Kindly ask for rebases if conflicts emerge, or if e.g. things changed in
+    CI that are causing a job to fail.
+*   Make sure PRs contain no fixup or merge commits before merging them. Merge
+    PRs if the button is green. Do not override branch protection.
+*   If the contributor does not appear to be making progress on the PR, mark
+    the PR as draft and add a comment similar to:
+
+    *I've marked this PR as draft while you work through the feedback in the
+    PR above. When you're ready to continue with review & testing of the PR,
+    please click "Ready for Review" at the bottom of the page.*
+
+2. Triaging Bugs
+^^^^^^^^^^^^^^^^
+
+Triaging bugs means interacting with the reporter until the issue is clear and
+can be labeled with the corresponding vertical.
+
+The main goal of this activity is identifying crucial issues that could impact
+users, and gathering more info so that we can drive towards solutions to those
+issues. We want to identify serious bugs and fix them.
+
+Handle these GitHub issues: `cilium/cilium issues that need Triager attention
+<https://github.com/cilium/cilium/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds%2Ftriage+-label%3Aarea%2Fdatapath+-label%3Aarea%2Fagent+-label%3Aarea%2Fhubble+-label%3Aarea%2FCI+-label%3Aarea%2FCI-improvement+-label%3Akind%2Ffeature+-label%3Akind%2Fenhancement+-label%3Akind%2Ftask>`_
+
+Issues are removed from this filter when they have the following labels added:
+
+*   ``area/datapath``
+*   ``area/agent``
+*   ``area/hubble``
+*   ``area/CI``
+*   ``area/CI-improvement``
+*   ``kind/feature``
+
+Issues are also removed from this filter when they have the following types
+set:
+
+*   Enhancement
+*   Task
+
+Please don't make changes to the filter without discussing in the `#tophat
+<https://slack.com/app_redirect?channel=tophat>`_ channel first.
```

Comment:
This link doesn't work.

Anyway in order to change this, someone would have to submit a PR to change this document so I think the change control is fine. We just need to make sure that this file has codeowners set to the right group so that the relevant people will look at it.